web/gui/dashboard_info.js

```diff
@@ -734,7 +734,23 @@ netdataDashboard.submenu = {
     },
 
     'ip.ecn': {
-        info: '<a href="https://en.wikipedia.org/wiki/Explicit_Congestion_Notification" target="_blank">Explicit Congestion Notification (ECN)</a> is a TCP extension that allows end-to-end notification of network congestion without dropping packets. ECN is an optional feature that may be used between two ECN-enabled endpoints when the underlying network infrastructure also supports it.'
+        info: '<a href="https://en.wikipedia.org/wiki/Explicit_Congestion_Notification" target="_blank">Explicit Congestion Notification (ECN)</a> '+
+        'is an extension to the IP and to the TCP that allows end-to-end notification of network congestion without dropping packets. '+
+        'ECN is an optional feature that may be used between two ECN-enabled endpoints when '+
+        'the underlying network infrastructure also supports it.'
+    },
+
+    'ip.multicast': {
+        info: '<a href="https://en.wikipedia.org/wiki/Multicast" target="_blank">IP multicast</a> is a technique for '+
+        'one-to-many communication over an IP network. '+
+        'Multicast uses network infrastructure efficiently by requiring the source to send a packet only once, '+
+        'even if it needs to be delivered to a large number of receivers. '+
+        'The nodes in the network take care of replicating the packet to reach multiple receivers only when necessary.'
+    },
+    'ip.broadcast': {
```

Comment:
Between `ip.ecn` and `ip.multicast` we have space, but for this chart description and next descriptions we do not have space between descriptions. I think we could add space between the descriptions to simplify code analysis.